src/activemasternode.cpp

```diff
@@ -154,29 +165,57 @@ void CActiveMasternode::ManageStateInitial()
         return;
     }
 
+    // Default to REMOTE
+    eType = MASTERNODE_REMOTE;
+
+    // Check if wallet funds are available
+    if(!pwalletMain) {
+        strNotCapableReason = "Wallet not available.";
+        LogPrintf("CActiveMasternode::ManageStateInitial -- not capable: %s\n", strNotCapableReason);
+        return;
+    }
+
+    if(pwalletMain->IsLocked()) {
+        strNotCapableReason = "Wallet is locked.";
+        LogPrintf("CActiveMasternode::ManageStateInitial -- not capable: %s\n", strNotCapableReason);
+        return;
+    }
+
+    if(pwalletMain->GetBalance() < 1000*COIN) {
```

Comment:
I didn't remove my comment - it's there but github shows that it's outdated for some reason.. Anyway, as I said I _though_ that wallet is needed for PS but probably I was wrong, because all the actual signing is done on clients and mn only combine everything together and broadcast. With that being said - let's try and see if it breaks smth :)